### PR TITLE
Add FEC Check

### PR DESCRIPTION
**Pull request by Mejiro-McQueen**
[SRGDS-23](https://jira.jpl.nasa.gov/browse/SRGDS-23?src=confmacro)

### CODE REVIEW

File: ait/dsn/plugins/AOS_FEC_Check.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+class AOS_FEC_Check(Plugin):
+    '''
+    Check if a AOS frame fails a Forward Error Correction Check
+    '''
+    def __init__(self, inputs=None, outputs=None, zmq_args=None, **kwargs):
+        super().__init__(inputs, outputs, zmq_args)
+        self.bytes_from_previous_frame = None
+        self.crc_func = crc_hqx
+        self.log_header = "AOS_FEC_CHECK =>"
```
Comment (cSchubes) [inline ait/dsn/plugins/AOS_FEC_Check.py:14]: amazing, fixes the issues we've had tying log messages to specific modules

**Review by cSchubes — APPROVED** [review]
Love it. Any plan to push this up to the AMMOS repo?

File: ait/dsn/plugins/AOS_FEC_Check.py (hunk 1 above)
Comment (Mejiro-McQueen) [reply ait/dsn/plugins/AOS_FEC_Check.py:14]: Yeah! 
Usually you would do:

`logger = logging.getLogger(__name__)`

Then your log header would be the actual function that's executing instead.

But...you know how it is...

**Comment by Mejiro-McQueen** [comment]
> Love it. Any plan to push this up to the AMMOS repo?

Not anytime soon, and especially not before we decide what to do when we fail the check.

**Pull request merged**